src/Setup/WixBA/WixBA.cs

```diff
@@ -61,6 +63,65 @@ public static void LaunchUrl(string uri)
         }
 
         /// <summary>
+        /// Open a log file.
+        /// </summary>
+        /// <param name="uri">URI to a log file.</param>
+        internal static void OpenLog(Uri uri)
+        {
+            // Switch the wait cursor since shellexec can take a second or so.
+            System.Windows.Input.Cursor cursor = WixBA.View.Cursor;
```

Comment:
I'm basically just saying this method and the one that follows can be refactored to share a private `ShellExecute` function to reduce code duplication.